settings.py

```diff
@@ -126,3 +133,12 @@
 #     "password ": "chaos",
 #     "db ": "db"
 # }
+
+SOURCE_PARTS_NAME = {
```

Comment:
I am not a Python expert :/

OrderedDict ensure iteration to follow the initial given order ?
Dict iteration is unpredictable ?